x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/eql/trial_license_complete_tier/eql.ts

```diff
@@ -9,7 +9,8 @@ import { v4 as uuidv4 } from 'uuid';
 import moment from 'moment';
 import supertestLib from 'supertest';
 import url from 'url';
-import expect from '@kbn/expect';
+import { default as kbnExpect } from '@kbn/expect';
+import expect from 'expect';
```

Comment:
We should migrate away from `@kbn/expect` towards the `expect` library: https://github.com/elastic/kibana/issues/215165